Makefile

```diff
@@ -132,13 +132,13 @@ test_python_property_verification:
 	$(PYTEST) -m "verification" -s tests/property
 
 test_python_optimized:
-	PYTHONOPTIMIZE=1 $(PYTEST) -m "not verification and not hypothesis" tests
+	PYTHONOPTIMIZE=1 $(PYTEST) -m "not verification and not hypothesis" tests/unit tests/integration tests/compilation tests/property
```

Comment:
I see. AFAICT the only remaining markers are in `tests/property` and `tests/tools/check_doc_tests.py`. The latter is now disabled in a target due to a misconfiguration (see https://github.com/Componolit/RecordFlux/pull/1265/files#r1042413100). IMO the mixture of test directories and markers that results from selecting tests via both mechanisms leads to a lot confusion about which target actually runs which part of the tests. I think the tradeoff of a bit more complex directory structure is worth the gained consistency if we remove the markers completely. What do you think?

Reply:
I agree, it would be better if we could get rid of these markers, so that there is only one mechanism. I think the differentiation between compilation and non-compilation tests is not relevant in `tests/tools/check_doc_test.py`, so we could just remove the markers there. For `tests/property` we need at least the differentiation between verification and non-verification tests to realize the `test_python_property` and `test_python_property_verification` make targets. What do you suggest?

Reply:
I would just split it into `property_verification` and `property` (or add `verification` and `non_verification` inside `property` but I really don't like `non_verification`).

Reply:
Fixed.